### PR TITLE
[fix] git info collection

### PR DESCRIPTION
**Pull request by devfox-se**
Added more checks when collecting git info. 
In case if one command fails its more likely others will fail too. In such case no data is collected and the process is stopped.

### CODE REVIEW

**Comment by devfox-se** [comment]
Covers #1538

File: aim/ext/utils.py [hunk 1]
```diff
@@ -23,23 +23,42 @@ def get_environment_variables():
...
+        ('git', 'rev-parse', '--abbrev-ref', 'HEAD'),
+        ('git', 'config', '--get', 'remote.origin.url'),
+        ('git', 'log', '--pretty=format:%h/%ad/%an', '--date=iso-strict', '-1'),
+    )
+    results = []
+    for cmd in cmds:
+        try:
+            r = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True)
+        except subprocess.CalledProcessError:
+            print('Something went wrong while collecting git info. Will skip this step.')
```
Comment (mihran113) [inline aim/ext/utils.py:50]: @devfox-se let's add a logger instead of basic print

**Review by alberttorosyan — APPROVED** [review]
lgtm

**Review by mihran113 — APPROVED** [review]
lgtm

**Pull request merged**